cg/store/api/find_business_data.py

```diff
@@ -94,6 +94,19 @@ def analyses_ready_for_vogue_upload(
 
         return records
 
+    def get_latest_nipt_analysis_to_upload(self):
```

Comment:
type hints

Reply:
These functions will be refactored immediately in my other pr.